crates/test-framework/src/spiced/mod.rs

```diff
@@ -24,29 +24,43 @@ use anyhow::Result;
 use flight_client::{Credentials, FlightClient};
 use spicepod::spec::SpicepodDefinition;
 use sysinfo::{Pid, ProcessesToUpdate, System};
-use tempfile::TempDir;
 
 use crate::utils::wait_until_true;
 
 pub struct SpicedInstance {
     child: Child,
-    tempdir: TempDir,
+    tempdir: PathBuf,
 }
 
 pub struct StartRequest {
     spiced_path: PathBuf,
     spicepod: SpicepodDefinition,
-    tempdir: TempDir,
+    tempdir: PathBuf,
     data_dir: Option<PathBuf>,
     prepared: bool,
 }
 
 impl StartRequest {
     pub fn new(spiced_path: PathBuf, spicepod: SpicepodDefinition) -> Result<Self> {
+        let tempdir = {
+            #[cfg(not(target_os = "windows"))]
+            {
+                let path = PathBuf::from("/tmp/spice_test_operator");
```

Comment:
Instead of hard coding a path, let's try to use filters: https://insta.rs/docs/filters/